UPGRADING.md

```diff
@@ -10,6 +10,18 @@ gem 'grape-swagger'
 gem 'grape-swagger-entity'
 ```
 
+`add_swagger_documentation` has changed from
+``` ruby
+  add_swagger_documentation \
+    api_version: 'v1'
+```
+to
+
+``` ruby
+  add_swagger_documentation \
+    version: 'v1'
```

Comment:
Can overwrite the API version sets by grape?
If yes, please write a spec to document it → :+1: